src/Haskell/Ide/Engine/Plugin/HieExtras.hs

```diff
@@ -139,6 +139,12 @@ mkModCompl label =
     Nothing Nothing Nothing Nothing hoogleQuery
   where hoogleQuery = Just $ toJSON $ "module:" <> label
 
+mkExtCompl :: T.Text -> J.CompletionItem
+mkExtCompl label =
+  J.CompletionItem label (Just J.CiModule) Nothing
```

Comment:
I think changing it to `CiKeyword` makes more sense, but if you think this is just bike shedding, then feel free to disregard :)

```suggestion
  J.CompletionItem label (Just J. CiKeyword) Nothing
```

Reply:
I agree, this does look better.